packages/contracts/contracts/votings/whitelist/WhitelistVotingFactory.sol

```diff
@@ -0,0 +1,54 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts/proxy/Clones.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "../../core/DAO.sol";
+import "./WhitelistVoting.sol";
+import "../../utils/Proxy.sol";
+import "../../plugin/IPluginFactory.sol";
+
+contract WhitelistVotingFactory is IPluginFactory {
+    using Address for address;
+    using Clones for address;
+
+    // TODO: Move it to MajorityVoting.
+    struct VoteConfig {
+        uint64 participationRequiredPct;
+        uint64 supportRequiredPct;
```

Comment:
Couldn't this be simpler in terms of a multisig? A single variable like `minSupportPct` seems to be able to do the job, unless we want to recycle the same struct within MajorityVoting